tools/mock_node/README.md

```diff
@@ -1,42 +1,43 @@
-# mock-network
+# mock-node
 This crate hosts libraries to start a test env for a single node by replacing the network module with a mock network environment. 
 The mock network environment simulates the interaction that the client will usually have with other nodes by 
 responding to the client's network messages and broadcasting new blocks. The mock network reads a pre-generated chain 
 history from storage.
 
-The crate has two files and another binary sits in ../bin/start_mock_network
+The crate has two files and a binary.
 
 ## mod.rs
 Implements `ChainHistoryAccess` and `MockPeerManagerActor`, which is the main 
 components of the mock network.
 ## setup.rs
 Provides functions for setting up a mock network from configs and home dirs.
-## start_mock_network.rs
+## main.rs
 A binary that starts a mock testing environment for ClientActor.
 It simulates the entire network by substituting PeerManagerActor with a mock network,
 responding to the client's network requests by reading from a pre-generated chain history
 in storage.
 
 ### Usage
 ```bash
-start_mock_network [FLAGS] [OPTIONS] <chain-history-home-dir> --mode <mode> [client-home-dir]
+start_mock_node [FLAGS] [OPTIONS] <chain-history-home-dir> --mode <mode> [client-home-dir]
```

Comment:
can you also add a section/command line example on how to compile it ? ( I guess now we no longer need to add features, right?)

Reply:
Yep I had that under notes, I'll make it more clear.